staging/src/k8s.io/code-generator/generate-internal-groups.sh

```diff
@@ -57,78 +55,60 @@ shift 5
 function codegen::join() { local IFS="$1"; shift; echo "$*"; }
 
 # enumerate group versions
-ALL_FQ_APIS=() # e.g. k8s.io/kubernetes/pkg/apis/apps k8s.io/api/apps/v1
-INT_FQ_APIS=() # e.g. k8s.io/kubernetes/pkg/apis/apps
-EXT_FQ_APIS=() # e.g. k8s.io/api/apps/v1
+FQ_APIS=() # e.g. k8s.io/kubernetes/pkg/apis/apps k8s.io/api/apps/v1
 for GVs in ${GROUPS_WITH_VERSIONS}; do
   IFS=: read -r G Vs <<<"${GVs}"
 
-  if [ -n "${INT_APIS_PKG}" ]; then
-    ALL_FQ_APIS+=("${INT_APIS_PKG}/${G}")
-    INT_FQ_APIS+=("${INT_APIS_PKG}/${G}")
-  fi
-
   # enumerate versions
   for V in ${Vs//,/ }; do
-    ALL_FQ_APIS+=("${EXT_APIS_PKG}/${G}/${V}")
-    EXT_FQ_APIS+=("${EXT_APIS_PKG}/${G}/${V}")
+    FQ_APIS+=("${APIS_PKG}/${G}/${V}")
   done
 done
 
 if [ "${GENS}" = "all" ] || grep -qw "deepcopy" <<<"${GENS}"; then
   echo "Generating deepcopy funcs"
   "${GOPATH}/bin/deepcopy-gen" \
-      --input-dirs "$(codegen::join , "${ALL_FQ_APIS[@]}")" -O zz_generated.deepcopy \
```

Comment:
deep-copy generation is still needed on internal types, so it's not clear how the internal package is getting added to the FQ_APIS list

defaulting, conversion, clients, listers, informers are only needed on external types

Reply:
ok, should we stop passing `deepcopy` to generate-internal-groups.sh everywhere then?

Reply:
The script, as it exists, will RE-DO the deepcopy  generation.  I wanted this PR to be just a small self-contained part of the larger PR - maybe easier if I just change the client-gen part and leave the rest for the subsequent commits?